content/foundation/governing-board-elections/2025/index.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Governing Board Elections 2025"
+template = "governing-board/elections.html"
+aliases = ["/governing-board/elections/2025"]
+extra.summary = """
+The list of candidates and their information.
+"""
+weight = 2
+# Hidden from navigation. The links are still accessible.
+# extra.navigation = false
++++
+
+Announcements are posted on our blog, our social channels, as well as in the [Office of the Matrix.org Foundation](https://matrix.to/#/#foundation-office:matrix.org) room, the [Office of the Governing Board](https://matrix.to/#/#governing-board-office:matrix.org) room, and the [Matrix News](https://matrix.to/#/#matrix-news:matrix.org) room.
+
+We encourage you to read our [membership page](/membership/) and announcements:
+
+* 2025-04 [Announcing the 2025 Governing Board elections](/blog/2025/04/election-announcement)
+
+Read on for our [election schedule](#schedule) and more information about the election process. We will be posting the list of candidates on this page at the end of the nomination period.
+
+## What is the Governing Board
+
+This page is focused on the 2025 election cycle. We encourage you to [learn more about the Governing Board itself here.](/foundation/governing-board/) 
+
+## Schedule
+
+* Announcement on Monday, April 28
+* Nominations from Saturday, May 3 to Friday, May 16
+* Campaigning from Saturday, May 17 to Friday, May 30
+* Voting from Saturday, May 31 to Friday, June 13
+* Results announced on Monday, June 16
+
+### Announcement (April 28)
+
+[We kickstarted the election process](/blog/2025/04/election-announcement/) in a blog post announcing the schedule, eligibility, and conditions of the vote.
+
+This year, we are running elections for the following constituency groups:
+
+* Gold Members
+* Silver Members
+* Individual Members
+* Associate Members
+
+### Nominations (May 3)
+
+You must be a member of one of the constituency groups in order to nominate yourself or someone else to be a candidate in the election.
+
+By far our largest constituency group are Individual Members, who number in the 100’s. If you have donated at least $60 USD to the Foundation since April 20, 2024 on [Donorbox](https://donorbox.org/keep-matrix-exciting), then you are eligible to nominate and vote. You must donate before the election period begins, as we cannot change voter rolls after that.
```

Comment:
Not blocking, but I thought it worth mentioning that the recent blog post made a point of pointing out that nominees must consent before being nominated (and having a good standing).

Reply:
Cosigned, and updated!